packages/web-app-files/src/components/Search/List.vue

```diff
@@ -173,7 +173,7 @@ export default defineComponent({
   watch: {
     searchResult: {
       handler: function () {
-        if (!this.searchResult) {
+        if (!this.searchResult || this.searchResult.totalResults === null) {
```

Comment:
Might break the oc10 search, did you tried it ?

Reply:
fck, no i haven't, but the ci 😢

Reply:
total results are the results that might exceed the limit, I think what you want is checking the searchResults.values.length

Reply:
Or just skip the Immediate 😂